db/seeds.rb

```diff
@@ -1,7 +1,20 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'csv'
+require_relative '../app/models/category'
+require_relative '../app/models/item'
+
+def import_product_csv
+  CSV.foreach('db/csv/first_seed.csv', headers: true, header_converters: :symbol) do |row|
+    Item.create(title: row[:title],
```

Comment:
To verify that every row has been created, can we make this a create! so that it will throw an error if the record isn't persisted? Alternatively, we could have a counter that puts the line count at the end for verification. I want to avoid the situation where it's not making items and we aren't sure why.